var/spack/repos/builtin/packages/quip/package.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import subprocess
+
+from spack.package import *
+
+
+class Quip(MakefilePackage):
+    """QUIP - QUantum mechanics and Interatomic Potentials
+
+    The QUIP package is a collection of software tools to carry out molecular
+    dynamics simulations. It implements a variety of interatomic potentials and
+    tight binding quantum mechanics. It can also call external packages or
+    serve as a plugin to packages such as LAMMPS, CP2K and the ASE Python
+    framework.
+    """
+
+    homepage = "https://libatoms.github.io/QUIP/"
+    url = "https://github.com/libAtoms/QUIP/archive/refs/tags/v0.9.14.tar.gz"
+    git = "https://github.com/libAtoms/QUIP.git"
+
+    # This package is a Fortran 95 package. It has hidden dependencies
+    # through the Fortran module files. Therefore running make in parallel
+    # will fail.
+    parallel = False
+
+    maintainers("hjjvandam")
+
+    license("GPL-2.0-only", checked_by="hjjvandam")
+
+    # We need to clone the GitHub repo for this library as the Makefile
+    # infrastructure tries to use `git` commands while building the code.
+    # Downloaded tar-balls are not `git` repos so building from those
+    # will fail.
+    version(
+        "0.9.14",
+        # sha256="b6c11c5199e8ba342ea138298fdc99b3861ccd7b9bf048dd95a1e8c125958e12",
```

Comment:
FWIW. The sha256 I get here is `587f7acce6c0538ec50f5d725db146f217fa16564e6851bc3f11e217cc048656`.